packages/auth/src/authClients/firebase.ts

```diff
@@ -1,7 +1,8 @@
+import { AuthClient } from './'
+
 import type FirebaseNamespace from 'firebase/app'
```

Comment:
This is Prettier's formatting, which ESLint doesn't like:
<img width="581" alt="Screen Shot 2021-05-14 at 12 21 27 PM" src="https://user-images.githubusercontent.com/2951/118318930-ebe69b00-b4ae-11eb-8213-f812e1d0d315.png">